src/renderer/components/ft-button/ft-button.js

```diff
@@ -15,7 +15,7 @@ export default Vue.extend({
   props: {
     label: {
       type: String,
-      required: true
+      required: false
```

Comment:
When `required` is false a default value should be set.  An empty string is fine in this case.  When a default value is given, adding `required: false` can be removed.